frontend/src/services/transaction.js

```diff
@@ -1,4 +1,4 @@
-import { connect } from 'get-starknet';
+import { connect } from 'starknetkit';
 import { CallData } from 'starknet';
```

Comment:
also, please remove `starknet` library as well, and use corresponding functionality from `starknetkit`

Reply:
Okay.

Reply:
still unresolved